WinUIGallery/ControlPages/BreadcrumbBarPage.xaml.cs

```diff
@@ -20,25 +20,38 @@ public BreadcrumbBarPage()
             this.InitializeComponent();
             BreadcrumbBar1.ItemsSource = new string[] { "Home", "Documents", "Design", "Northwind", "Images", "Folder1", "Folder2", "Folder3" };
 
-            BreadcrumbBar2.ItemsSource = new ObservableCollection<Folder>(folders);
+
+            BreadcrumbBar2.ItemsSource = new ObservableCollection<Folder>{
```

Comment:
Why does is the Folder ObservableCollection declared here? Can we not reference the List that is declared at the top?